setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='prismic',
-    version='1.4.2',
+    version='1.4.3',
```

Comment:
Could you upgrade the minor version instead the patch version?
Otherwise, your PR is OK :)
Again, thanks.